docs/examples/Naive_Classification.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
The notebook needs to have a broader, more informative title. Why not use the one above, or something similar: "using a Hidden Markov Model for classification", perhaps?

Forward algorithm isn't specific enough (there are a number of forward algorithms). A bit of explanatory text, below the title, saying what we're up to and what this forward algorithm is being used for.